app/controllers/users_controller.rb

```diff
@@ -13,7 +13,8 @@ def update
 
   def show
     @user = current_user
-#   @todo = @user.todo.visible_to(current_user) 
+    @items = @user.items
```

Comment:
I am a bit confused here. We usually use @user = User.find(params[:id]) but in this case, when I try and use that instead of @user = current_user, then I get an error that says it cannot find a User with id=' '